### PR TITLE
Gm handbook lmao

### PR DESCRIPTION
**Pull request by Arikatsu**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.

## Type of changes

- [x] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/handbook/src/ui/components/VirtualizedGrid.tsx [hunk 1]
```diff
@@ -64,8 +66,21 @@ class VirtualizedGrid<T> extends React.Component<IProps<T>, IState> {
...
+    componentDidMount() {
+        this.setState({
+            itemsPerRow: Math.floor((window.innerWidth - 650) / (this.props.itemHeight + (this.props.gap ?? 0)))
+        });
+
+        window.addEventListener("resize", () => {
+            this.setState({
+                itemsPerRow: Math.floor((window.innerWidth - 650) / (this.props.itemHeight + (this.props.gap ?? 0)))
+            });
+            console.log(window.innerWidth)
```
Comment (KingRainbow44) [inline src/handbook/src/ui/components/VirtualizedGrid.tsx]:
```suggestion
```

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]
also run spotless apply

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]
address changes sent via discord. changes include:
- removing random white box on home screen
- fixing text scaling on item and entity names
- fixing the position of the scroll bar to not cut it off
- fixing text scaling on home page text, 'achievements' gets cut off
- add purple hover effect on discord community
- remove scroll bars present due to small height on command/card text boxes
- import font and icons via react, these need to be seen at compile time

**Comment by KingRainbow44** [comment]
image
image
image
image
image
image
image

**Comment by lilmayofuksu** [comment]
goobers

**Comment by KiminaNatsuki** [comment]
image

**Comment by Arikatsu** [comment]
goobers

**Comment by Arikatsu** [comment]
this pr is so cursed 💀

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]
fix your entity icon

**Review by KingRainbow44 — APPROVED** [review]

**Pull request merged**